server/index-node/src/schema.graphql

```diff
@@ -7,6 +7,12 @@ scalar String
 type Query {
   indexingStatusesForSubgraphName(subgraphName: String!): [SubgraphIndexingStatus!]!
   indexingStatuses(subgraphs: [String!]): [SubgraphIndexingStatus!]!
+  proofOfIndexing(request: ProofOfIndexingInput!): Bytes
+}
+
+input ProofOfIndexingInput {
+  subgraphName: String!
```

Comment:
`subgraph` is fine, but it should be the ID of the subgraph deployment, not the name of a subgraph.